portality/static/doaj/js/available_facetviews/admin.background_jobs.facetview.js

```diff
@@ -45,6 +45,9 @@ jQuery(document).ready(function($) {
             paramsBlock += "<strong>Parameters:</strong><br>";
             for (var key in resultobj.params) {
                 var val = resultobj.params[key];
+                if (val instanceof Array) {
```

Comment:
I've never used instanceof before, so I'm not sure of the compatibility of it.  There is a $.isArray function in jquery, which makes me suspicious of instanceof.  Anyone know?  A quick google hasn't turned up anything obviously pro or against this approach.